package.json

```diff
@@ -9,9 +9,10 @@
   "scripts": {
     "start": "webpack-dev-server --open",
     "build": "webpack",
-    "test": "mocha --opts test/mocha.opts test/**/*.spec.js",
+    "test": "yarn lint && yarn lint:styles && yarn test:unit",
+    "test:unit": "mocha --opts test/mocha.opts test/**/*.spec.js",
     "lint": "eslint src/**/*.js src/**/*.jsx",
-    "lint:styles": "stylelint src/**/*.css"
+    "lint:styles": "stylelint src/**/*.css --allow-empty-input && stylelint src/**/*.scss --allow-empty-input"
```

Comment:
Just want to keep the support of both, but the main plan should be having only scss files